### PR TITLE
[7.52.x] DROOLS-6268 : Guided Rule Editor: Add memberOf and not memberOf operators

### PR DESCRIPTION
**Pull request by Rikkola**
(cherry picked from commit 5c4563d7533cc9eefcad61e9e7387f1c36ffc3f1)

**JIRA**: 

[DROOLS-6268](https://issues.redhat.com/browse/DROOLS-6268)

**referenced Pull Requests**:

* https://github.com/kiegroup/kie-soup/pull/194
* https://github.com/kiegroup/drools/pull/3689
* https://github.com/kiegroup/kie-wb-common/pull/3620
* https://github.com/kiegroup/drools-wb/pull/1483

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* Retest PR: <b>jenkins retest this</b>
* A full downstream build: <b>jenkins do fdb</b>
* A compile downstream build: <b>jenkins do cdb</b>
* A full production downstream build: <b>jenkins do product fdb</b>
* An upstream build: <b>jenkins do upstream</b>
</details>

### CODE REVIEW

File: kie-wb-common-widgets/kie-wb-common-ui/src/test/java/org/kie/workbench/common/widgets/client/datamodel/PackageDataModelOracleCompletionsTest.java [hunk 1]
```diff
@@ -356,7 +356,7 @@ public void testFactFieldConnectiveOperatorCompletions() {
         Callback<String[]> personThisCallback = spy(new Callback<String[]>() {
             @Override
             public void callback(final String[] personThisConnectiveOperators) {
-                assertEquals(3,
+                assertEquals(5,
```
Comment (jomarko) [inline kie-wb-common-widgets/kie-wb-common-ui/src/test/java/org/kie/workbench/common/widgets/client/datamodel/PackageDataModelOracleCompletionsTest.java:359]: should we check "memberOf" and "not memberOf" were added?

File: kie-wb-common-widgets/kie-wb-common-ui/src/test/java/org/kie/workbench/common/widgets/client/datamodel/PackageDataModelOracleCompletionsTest.java [hunk 2]
```diff
@@ -445,7 +445,7 @@ public void callback(final String[] personRankConnectiveOperators) {
         Callback<String[]> personNameCallback2 = spy(new Callback<String[]>() {
             @Override
             public void callback(final String[] personNameConnectiveOperators) {
-                assertEquals(13,
+                assertEquals(15,
```
Comment (jomarko) [inline kie-wb-common-widgets/kie-wb-common-ui/src/test/java/org/kie/workbench/common/widgets/client/datamodel/PackageDataModelOracleCompletionsTest.java:448]: should we check "memberOf" and "not memberOf" were added?

File: kie-wb-common-widgets/kie-wb-common-ui/src/test/java/org/kie/workbench/common/widgets/client/datamodel/PackageDataModelOracleCompletionsTest.java (hunk 1 above)
Comment (Rikkola) [reply kie-wb-common-widgets/kie-wb-common-ui/src/test/java/org/kie/workbench/common/widgets/client/datamodel/PackageDataModelOracleCompletionsTest.java:359]: We can, adding that.

Comment (Rikkola) [reply kie-wb-common-widgets/kie-wb-common-ui/src/test/java/org/kie/workbench/common/widgets/client/datamodel/PackageDataModelOracleCompletionsTest.java:359]: @jomarko Updated

**Review by jomarko — APPROVED** [review]
thank you @Rikkola

**Comment by mareknovotny** [comment]
closing as the jira was retargeted only to master branch

**Review by karreiro — APPROVED** [review]

**Comment by mareknovotny** [comment]
jenkins retest this please

**Pull request merged**